### ISSUE TITLE
Chat say and say_team is not displayed in demos

### ISSUE THREAD

**Issue by crashzk** [Hexer10/HexTags#94]
**Describe the Bug**
Chat say and say_team is not displayed in demos.

**To Reproduce**
1. Install HexTags version **Simple Chat Processor** - https://github.com/Hexer10/HexTags/pull/90
2. Send messages in chat, either **`say`** or **`say_team`**

**Expected Behavior**
In the recorded server demos, all messages sent in the chat, either by **`say`** or **`say_team`** must be recorded in the demo and then it will be possible to view the messages if someone watches the demo.

**System Details**
 - **OS:** Ubuntu
 - **Game:** CSGO
 - **Sourcemod Version:** 1.11.0.6930, jit-x86 (build 1.11.0.6930)
 - **Sourcemod Plugins List:** https://gist.github.com/crashzk/05fbaa7d1d773d1395a455c59a58827a
 - **Your hextags.cfg:** 
 ```
 "HexTags"
{
	"@DONO"
	{
		"ChatTag"	"{darkred}[ DONO ] "
		"ChatColor"	"{green}"
		"NameColor" "{teamcolor}"
	}

	"@SUBDONO"
	{
		"ChatTag"	"{darkblue}[ SUB-DONO ] "
		"ChatColor"	"{green}"
		"NameColor" "{teamcolor}"
	}

	"@ADMIN"
	{
		"ChatTag"	"{green}[ ADMIN ] "
		"ChatColor"	"{green}"
		"NameColor" "{teamcolor}"
	}

	"@MOD"
	{
		"ChatTag"	"{orchid}[ MOD ] "
		"ChatColor"	"{green}"
		"NameColor" "{teamcolor}"
	}

	"@MODTRIAL"
	{
		"ChatTag"	"{orchid}[ MOD TRIAL ] "
		"ChatColor"	"{green}"
		"NameColor" "{teamcolor}"
	}

	"@PREMIUM"
	{
		"ScoreTag"	"[ PREMIUM ]"
		"ChatTag"	"{gold}[ PREMIUM ] "
		"ChatColor"	"{green}"
		"NameColor" "{teamcolor}"
	}

	"@VIP"
	{
		"ScoreTag"	"[ VIP ]"
		"ChatTag"	"{gold}[ VIP ] "
		"ChatColor"	"{green}"
		"NameColor" "{teamcolor}"
	}

	"Default"
	{
		"ChatTag"	"{darkred}i ♥ zk "
		"ChatColor"	"{default}"
		"NameColor" "{teamcolor}"
	}
}
```

**Additional Context**
To register the chat in the demo, I configured the GOTV with the following configuration in the **`server.cfg`** file:
```
//GOTV
tv_enable "1" 						
tv_autorecord "1"						
tv_delay "10" 							
tv_advertise_watchable "1"
tv_maxclients "32"
tv_title "GOTV" 
tv_name "GOTV"
tv_password ""						
tv_allow_camera_man "1" 		
tv_allow_static_shots "1" 		
tv_snapshotrate "32" 					
tv_nochat "0" 						
tv_timeout "45" 						
tv_port "27015"						
tv_delaymapchange "0"					
tv_dispatchmode "1"					
tv_relayradio "1"					
tv_relayvoice "1"				
tv_relaytextchat "2"					
tv_transmitall "1"						
```
After everything is configured, just enter and send messages, as you can see later in the demo, the chat messages are not displayed, only voice and radio.

**Comment by crashzk** [comment]
Just for an update, version with **[Chat Processor](https://forums.alliedmods.net/showthread.php?p=2448733)** support also has this same bug, I just tested it here.

**Comment by Hexer10** [comment]
Do you have any other plugins that could be interfering? I just tried with viewing chat messages in the demo first with only chat processor and then with hextags as well, and both times I was able to see the messages in the demo.

**Comment by crashzk** [comment]
Which makes changes to the chat only Hextags. In any case, I did a more in-depth test, I tested another Tags and Chats configuration plugin, it also uses Simple Chat Processor, same problem, the demo does not show the chat, I completely removed the plugin and it shows the chat in the demo.

I've already tested plugins that use the Simple Chat Processor and Chat Processor versions, both have the same problem. It even seems like it would be something in these plugins, some new feature in CSGO maybe?

Anyway, I'm going to test further, for now I completely removed the HexTags and the Simple Chat Processor, test without them to see if the chat is normal, anything I'll return here.
